proto/internal/temporal/server/api/replication/v1/message.proto

```diff
@@ -194,3 +195,18 @@ message SyncHSMAttributes {
     temporal.server.api.history.v1.VersionHistory version_history = 4;
     temporal.server.api.persistence.v1.StateMachineNode state_machine_node = 5;
 }
+
+message BackfillHistoryTaskAttributes {
+    string namespace_id = 1;
+    string workflow_id = 2;
+    string run_id = 3;
+
+    temporal.server.api.persistence.v1.VersionedTransition versioned_transition = 4;
+    repeated temporal.server.api.history.v1.VersionHistoryItem version_history_items = 5;
+    temporal.server.api.workflow.v1.BaseExecutionInfo base_execution_info = 6;
+    repeated temporal.api.common.v1.DataBlob event_batches = 7;
+
+    string new_run_id = 8;
+    temporal.api.common.v1.DataBlob new_run_event_batch = 9;
```

Comment:
nit: maybe we should consider defining a separate message for those new run attributes so they can be reused by other tasks as well.

Reply:
added new message.